### DISCUSSION TITLE
new user.. adding deault filter list help

### DISCUSSION THREAD

**Discussion by taffybo** [uBlockOrigin/uAssets#22966] [Q&A] [answered]
![Screenshot 2024-03-18 115908](https://github.com/uBlockOrigin/uAssets/assets/163846615/9e53a116-e579-40eb-9de8-256f364ca937)

Hi, how do I go about seeing this list 
Thanks

**Comment by ghajini** [comment, marked as answer]
you don't have to do anything upon installing ublock origin
all default lists are updated automatically after a while

**Comment by ghost** [comment]
https://github.com/gorhill/uBlock/wiki/Dashboard

**Comment by taffybo** [comment]
How do I tick or untick boxes as shown in pic
